### ISSUE TITLE
Is there any way to use it in on prem hosted apigee instances?

### ISSUE THREAD

**Issue by susamn** [darumatic/apigee-cli#4]
This looks like a great tool. Can I use it for on prem apigee instances? It looks like you are only having provision for SaaS instances

**Comment by mdelotavo** [comment]
> It looks like you are only having provision for SaaS instances
We only really use the SaaS product, so we never had to support on-premises installations.

I'm not exactly sure all the steps that need to be done to support on-prem but changing `APIGEE_ADMIN_API_URL = 'https://api.enterprise.apigee.com'` under `apigee/__init__.py` might be a good start.

This might also help:
https://www.googlecloudcommunity.com/gc/Apigee/Securely-expose-Management-API-for-on-premises-installation-of/td-p/20895

In other words, to support on-prem, you would probably have to fork and make some changes to the source code and test the changes from your end.
